logstash-input-jmx.gemspec

```diff
@@ -27,4 +27,3 @@ Gem::Specification.new do |s|
   s.add_development_dependency 'logstash-devutils'
   s.add_development_dependency 'logstash-codec-plain'
 end
-
```

Comment:
Not sure if a trailing line is required, but I would not remove it. If your editor is set up to remove empty lines at the end of files, I'd change that setting because some files (like asciidoc) might get messed up if you remove  empty lines.

Reply:
Oh weird. At any rate, I wouldn't arbitrarily remove empty lines, so if your editor is set up to do that, you should change the setting.

Reply:
I use Atom, and I disabled the Whitespace package (Preferences > Packages-> Whitespace) as you suggested.  Best I can tell, the extra line at the end is present in all three files.